components/bl-sweet-alert/README.md

```diff
@@ -0,0 +1,181 @@
+| Grow <br> `grow`                              | *Select* <br> [`row`, `column`, `fullscreen`, `none`]                                                                         | `none`                                                                                                                                                                                                            |                           | NO           | YES        | Controls the direction the popup should grow in. Paired with window position.                                                                                                                             |
+| Additional Settings <br> `additionalSettings` | *JSON*                                                                                                                        | `{ "showClass": { "popup": "swal2-show", "backdrop": "swal2-backdrop-show", "icon": "swal2-icon-show" }, "hideClass": { "popup": "swal2-hide", "backdrop\": "swal2-backdrop-hide", "icon": "swal2-icon-hide" } }` | Additional Settings Logic | YES          | YES        | This logic allows to specify the additional popup settings.                                                                                                                                               |
+
+## Default Buttons Settings
+
+```
+{
+  "showConfirmButton": true,
+  "confirmButtonText": "OK",
+  "confirmButtonColor": "#3085d6",
+  "confirmButtonAriaLabel": "",
+  "focusConfirm": true,
+  "showDenyButton": false,
+  "denyButtonText": "No",
+  "denyButtonColor": "#dd6b55",
+  "denyButtonAriaLabel": "",
+  "focusDeny": false,
+  "showCancelButton": false,
+  "cancelButtonText": "Cancel",
+  "cancelButtonColor": "#aaa",
+  "cancelButtonAriaLabel": "",
+  "focusCancel": false,
+  "showCloseButton": false,
+  "closeButtonHtml": "&times;",
+  "closeButtonAriaLabel": "Close this dialog"
+}
+```
+
+## Events
+
+| Name       | Triggers                                                             | Context Blocks                                                                                          |
+|------------|----------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------|
+| On Confirm | when the "Confirm" button was clicked.                               | response: `{ isConfirmed: Boolean, isDenied: Boolean, isDismissed: Boolean, value: Boolean \| String }` |
```

Comment:
Do we realy need `isDenied` and `isDismissed` at On Confirm event? Same for others.